frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -136,7 +160,8 @@ const AdminDashboard = (): React.ReactElement => {
         name: "actions",
         label: " ",
         options: {
-          customBodyRender: () => {
+          // eslint-disable-next-line
+          customBodyRender: (value, tableMeta, updateValue) => {
```

Comment:
instead of disabling the lint, can you add the types to these variables? 
<img width="791" alt="image" src="https://user-images.githubusercontent.com/57159970/158081183-46ca3f63-6479-4308-80cc-b0c4d03659fa.png">

Reply:
The lint disabling was because the `value` and `updateValue` are unused variables but I need to include them to not get an error when using `tableMeta` for some reason.